bin/commands/countly.sh

```diff
@@ -10,7 +10,20 @@ done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
 #get current countly version
-VERSION="$(grep -oP 'version:\s*"\K[0-9\.]*' "$DIR/../../frontend/express/version.info.js")"
+VERSION="$(egrep -o 'version: *"[0-9\.]+"' "$DIR/../../frontend/express/version.info.js" | sed -E -e 's,^version: *"([0-9\.]+)"$,\1,')"
+
+#Activate our nodejs, if using nodeenv
```

Comment:
I'm using nodeenv on macOS since it provides a nice, clean environment for Count.ly without affecting any other NodeJS applications that may be running. I suggest switching over to nodeenv since Count.ly isn't using the latest NodeJS.

Reply:
In overall nodeenv is useless to Countly, since in production we have control over what is installed and nothing else should be running and preconfigured on the server. It will only be one more dependency to worry about when making offline builts with prebuild binaries, etc
But since here it is optional, should be ok, only not sure about the path, seems to be tailored to your system? Is there some default path to be used in such cases?

Reply:
I'm running a production environment, with multiple versions of NodeJS required for other various low-volume services (most run in 12.x, but a few are running on 14.x). I suppose I could run either Count.ly or those other services in Docker containers, but that adds an overhead that is disproportionate to the load required.
The biggest problem is that Count.ly is currently now *several* versions behind on NodeJS - requiring 10.x while the most recent LTS is 12.x and release is 14.x.